app/src/main/java/org/gdg/frisbee/android/api/OkClientProvider.java

```diff
@@ -0,0 +1,45 @@
+package org.gdg.frisbee.android.api;
+
+import android.content.Context;
+
+import com.squareup.okhttp.Cache;
+import com.squareup.okhttp.OkHttpClient;
+
+import java.io.File;
+
+import retrofit.client.Client;
+import retrofit.client.OkClient;
+
+/**
+ * Created by <a href="mailto:marcusandreog@gmail.com">Marcus Gabilheri</a>
+ *
+ * @author Marcus Gabilheri
+ * @version 1.0
+ * @since 7/26/15.
+ */
+public final class OkClientProvider {
+
+    // Cache size for the OkHttpClient
+    private static final int DISK_CACHE_SIZE = 50 * 1024 * 1024; // 50MB
+
+    private static OkClient okClient;
+
+    private OkClientProvider() {
+    }
+
+    public static OkHttpClient getOkHttpClient(Context context) {
+        OkHttpClient okHttpClient = new OkHttpClient();
+        // Install an HTTP cache in the application cache directory.
+        File cacheDir = new File(context.getApplicationContext().getCacheDir(), "http");
+        Cache cache = new Cache(cacheDir, DISK_CACHE_SIZE);
```

Comment:
Ok! Sounds good. So let's see if I understood it. Since the OkClient() is just a wrapper for the OkHttpClient. Should we have just the OkHttpClient be a wrapper and them when we set the client for a RestAdapter we do something like this:
```java
restAdapter.setClient(new OkClient(App.getInstance().getOkHttpClient());
```

Does that sounds good? Or should we have a separate singleton to hold a single OkClient?